grow/server/manager.py

```diff
@@ -83,6 +83,7 @@ def start(pod, host=None, port=None, open_browser=False, debug=False,
             serving.run_simple(host, port, app, request_handler=handler, threaded=True)
             done = True
         except socket.error as e:
+            # if any(x in str(e) for x in ('Errno 48', 'Errno 98')):
```

Comment:
you should maybe do a regular for loop here since this'll evaluate each of the expressions even if the first one returns `true`

Reply:
I agree that it makes it evaluate each string check, but looking at how the other logic would need to change, it makes it more complicated to split it into a normal loop.

Reply:
alright, since N is always going to be small here, i don't mind this pattern here, but in general we should prefer writing optimal code where possible, otherwise small memory or cpu issues can seem trivial at first but snowball into a mess in the future.